plug.vim

```diff
@@ -2185,7 +2218,9 @@ endfunction
 
 function! s:rm_rf(dir)
   if isdirectory(a:dir)
-    call s:system((s:is_win ? 'rmdir /S /Q ' : 'rm -rf ') . plug#shellescape(a:dir))
+    call s:system(s:is_win
+    \ ? 'rmdir /S /Q '.plug#shellescape(a:dir)
```

Comment:
Should we change this one to an array as well?

Reply:
`rmdir` is a cmd.exe builtin command. I don't know a default external program for deleting files/directories on Windows. If there is a dll for this, then we can replace this line with `['rundll32.exe', 'delete.dll,DeleteProtocol', a:dir]`.